### ISSUE TITLE
refactor: remove `wrapperd` in favor launcher

### ISSUE THREAD

**Issue by smira** [siderolabs/talos#9472]
Experiment, remove `wrapperd` completely replacing it with https://pkg.go.dev/kernel.org/pub/linux/libs/security/libcap/cap#NewLauncher

The idea is that we should be able to set most (if not all) of wrapperd stuff by using Launcher.

This should cut our latency to fork/exec child programs.

**Comment by dsseng** [comment]
As far as I understand, we cannot set ctty from our callback context before fork-exec, so we would likely need to fork `kernel.org/pub/linux/libs/security/libcap/cap` (GPL-2.0-only or BSD licensed) to implement ctty by modifying SysProcAttr. Apparently we use this feature to set controlling TTY for dashboard process.

**Comment by dsseng** [comment]
Nvm, apparently *pa is given in the callback. Let me try that tomorrow. Also, do we have tests for dropping capabilities? I believe I should create one, just to be sure and test my changes.

`cat /proc/1324/status | grep Cap` and `/sbin/capsh --decode=000001f3fdfeffff` can be used to get and decode capability sets, respectively, so using API to get processes and then get their caps from sysfs should work just fine.

**Comment by smira** [comment]
> Nvm, apparently *pa is given in the callback. Let me try that tomorrow. Also, do we have tests for dropping capabilities? I believe I should create one, just to be sure and test my changes.
> 
> `cat /proc/1324/status | grep Cap` and `/sbin/capsh --decode=000001f3fdfeffff` can be used to get and decode capability sets, respectively, so using API to get processes and then get their caps from sysfs should work just fine.

yes, in the integration test we could do `talosctl read` API equivalent and compare to the well known capability value e.g. for the `containerd` process(es). (as everything inherits from them)
